frontend/src/components/maps/leaflet/InfoSlideOut/InfoSlideOut.test.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import 'leaflet';
+import { Map as LeafletMap } from 'leaflet';
+import { Map as ReactLeafletMap, MapProps } from 'react-leaflet';
+import Adapter from 'enzyme-adapter-react-16';
+import Enzyme, { mount } from 'enzyme';
+import InfoSlideOut from './InfoSlideOut';
+import 'leaflet/dist/leaflet.css';
+import { waitFor } from '@testing-library/dom';
+import { Button } from 'react-bootstrap';
+
+jest.mock('axios');
+Enzyme.configure({ adapter: new Adapter() });
+
+let mapRef: React.RefObject<ReactLeafletMap<MapProps, LeafletMap>> | undefined;
+
+const MapComponent = () => {
+  const [open, setOpen] = React.useState(false);
+  mapRef = React.useRef<any>();
+  return (
+    <div id="mapid" style={{ width: 500, height: 500 }}>
+      <ReactLeafletMap ref={mapRef} center={[48.423078, -123.360956]} zoom={18}>
+        <InfoSlideOut open={open} setOpen={() => setOpen(!open)} />
+      </ReactLeafletMap>
+    </div>
+  );
+};
+
+describe('InfoSlideOut View', () => {
+  beforeEach(() => {
+    mapRef = undefined;
+  });
+
+  it('Should render the slide out button', () => {
+    const component = mount(<MapComponent />);
+    waitFor(() => expect(mapRef?.current).toBeTruthy(), { timeout: 500 });
+    const infoButton = component.find(Button).first();
+    expect(infoButton).toBeDefined();
+    expect(infoButton.props().id).toBe('slideOutInfoButton');
+  });
+
+  it('Should component to be closed by default', () => {
```

Comment:
nit: Component should be closed by default

Reply:
woops